components/new-contribution-flow/constants.js

```diff
@@ -4,3 +4,19 @@ export const STEPS = {
   PAYMENT: 'payment',
   SUMMARY: 'summary',
 };
+
+export const PAYMENT_METHOD_TYPES = {
```

Comment:
These `PAYMENT_METHOD_TYPES` are not specific to the contribution flow, can we rather put that in `lib/constants/payment-methods.js`?

Reply:
These conversions are a bit confusing to me. Ie, why do we need that?

`STRIPE: 'CREDIT_CARD',`

Can't we write that directly? 

`const hostHasStripe = supportedPaymentMethods.includes(PAYMENT_METHOD_TYPES.CREDIT_CARD);`

In the end, I would only move that as constants if there was a clear definition of what the keys are and what the matching values are.

Reply:
For sure, I think I made the constants because that's the pattern on the API but then didn't end up using them a lot.... let me clean those up

Reply:
@Betree the ones in `lib/constants` are lower case and the API returns upper case like `CREDIT_CARD` in this particular field we're reading so we need some constants specifically for this

@znarf I have removed the name changes and unused constants

Reply:
@sbinlondon You can still add the new constants to the file as a new object, it's not a problem to export multiple objects from there.

These are values returned by the API that are meant to be used in multiple places of the frontend (ie. on the settings page to edit your payment methods). It makes sense to have it in a shared library rather than a subfolder.

Reply:
If the names conflict, you can name this one like `GQLV2_PAYMENT_METHOD_TYPES`

Reply:
Great idea, I was thinking too 'in the box', you're right they should go in the main constants file. Have moved them and named them accordingly!